src/Sulu/Component/Webspace/Analyzer/Attributes/PortalInformationRequestProcessor.php

```diff
@@ -55,8 +51,14 @@ public function process(Request $request, RequestAttributes $requestAttributes)
         }
 
         $attributes['localization'] = $portalInformation->getLocalization();
+
+        if (!$attributes['localization'] && $portalInformation->getPortal()) {
+            $attributes['localization'] = $portalInformation->getPortal()->getDefaultLocalization();
+        }
+
         if ($attributes['localization']) {
```

Comment:
here I'm not sure if i should set also `locale` always e.g.:

```
if ($attributes['localization']) {
    $request->setLocale($attributes['localization']->getLocale());
}

$attributes['locale'] = $request->getLocale();
```

what you think @danrot